web/satellite/src/components/navigation/NavigationArea.vue

```diff
@@ -255,6 +256,7 @@ export default class NavigationArea extends Vue {
     public navigateToBuckets(): void {
         this.analytics.eventTriggered(AnalyticsEvent.UPLOAD_IN_WEB_CLICKED);
         this.closeDropdowns();
+        this.analytics.pageVisit(RouteConfig.Buckets.with(RouteConfig.BucketsManagement).path);
```

Comment:
why not RouteConfig.Buckets.path like the line below?

Reply:
because when I write this ("RouteConfig.Buckets.path") in pageVisit , it sends this link "/buckets" instead of "/buckets/management"